src/dvc_objects/db.py

```diff
@@ -42,6 +43,18 @@ def __eq__(self, other):
     def __hash__(self):
         return hash((self.fs.protocol, self.path))
 
+    def _init(self):
+        if self.read_only:
+            return
+
+        if self._initialized:
+            return
+
+        for num in range(0, 256):
```

Comment:
Obviously it should be not 256, but 16**PREFIX_LEN, but it is hardcoded in other places, so I didn't bother for now.